src/panels/config/integrations/ha-integration-card.ts

```diff
@@ -240,6 +266,13 @@ export class HaIntegrationCard extends LitElement {
     `;
   }
 
+  private async _fetchManifests() {
+    const manifests = {};
+    const fetched = await fetchIntegrationManifests(this.hass);
```

Comment:
We should move this fetching up to the parent. Now each card will fetch all manifests and just extract one.